### PR TITLE
some AlphaThickSegmentComputer fixes

### PR DESCRIPTION
**Pull request by kerautret**
some small improvements:
fix segment display errors which could appear when displaying a small segment. Fix also a non initialized attribute with some improvements on bounding box computation.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# DGtal 0.9.2
+
+
+## Bug Fixes
+- *Geometry Package*
+ - AlphaThickSegmentComputer: fix segment display errors which could appear
+   when displaying a small segment. Fix a non initialized attribute with
+   some improvements on bounding box computation with orientation check.
+   (B. Kerautret, [#1123](https://github.com/DGtal-team/DGtal/pull/1106))
```
Comment (dcoeurjo) [inline ChangeLog.md]: Bertrand ?
PR numbers do not match;)

File: src/DGtal/geometry/curves/AlphaThickSegmentComputer.ih [hunk 2]
```diff
@@ -685,7 +712,11 @@ DGtal::AlphaThickSegmentComputer< TInputPoint, TConstIterator>::projectOnStraigh
   
   ptProjected[0]= ptA[0]+vABn[0]*(distPtA_Proj);
   ptProjected[1] = ptA[1]+vABn[1]*(distPtA_Proj);  
-  return  (distPtA_Proj>=0 && (ptProjected-ptA).norm() <= vAB.norm()); 
+  
+  return  distPtA_Proj>=0 && ((ptA[0]<ptB[0] && ptProjected[0]<=ptB[0] ) ||
```
Comment (dcoeurjo) [inline src/DGtal/geometry/curves/AlphaThickSegmentComputer.ih]: :ok_hand:

**Comment by kerautret** [comment]
@dcoeurjo done thanks

**Comment by kerautret** [comment]
@dcoeurjo can we merge ?

**Comment by dcoeurjo** [comment]
done;)thx

**Pull request merged**